src/BloomExe/Publish/PublishView.cs

```diff
@@ -509,8 +509,11 @@ public void SetDisplayMode(PublishModel.DisplayModes displayMode)
 					_pdfViewer.Visible = false;
 					Cursor = Cursors.WaitCursor;
 
-					// Calls LoadBookIfNeeded() which calls BringBookUpToDate() and Save() if needed
-					_model.DoAnyNeededAudioCompression();
+					// Calls BringBookUpToDate() and Save() if needed.
+					// Not sure if this is necessary at this point; it replaced a call to DoAnyNeededAudioCompression(),
+					// which is NOT wanted at this point since it can be time-consuming, and the old comment
+					// emphasized that doing the compression involved this step, so it seemed safer to keep this.
+					_model.LoadBookIfNeeded();
```

Comment:
Hmmm; seems at first glance like something that could easily go in the new SendBook method? In addition to feeling logical there in that file-manipulate code (and here, a place that is otheriwise all about *view*ing), we want the UI here to come up as quickly as possible.